elastic/kibana
# [ResponseOps] Removing usage of scripted_metric within the actions telemetry

Resolves https://github.com/elastic/kibana/issues/181850

## Summary
The Elasticsearch team is looking at limiting the usage of `scripted_metric` within serverless, and this PR updates the telemetry code that uses scripted_metric within the actions plugin.


### Checklist



### To verify
The best way to verify this PR is to verify that the telemetry functional tests pass. The data should be reported the same as it was before this change.

Here are the list of fields that were calculated using `scripted_metric`:
- count_total
- count_by_type
- count_gen_ai_provider_types
- count_active_total
- count_active_by_type
-  count_actions_executions_per_day
- count_actions_executions_by_type_per_day
- count_actions_executions_failed_per_day
- count_actions_executions_failed_by_type_per_day

If you'd like to run the telemetry locally:
- Create a couple of rules with different actions
- Change actions telemetry task [schedule](https://github.com/elastic/kibana/blob/main/x-pack/plugins/actions/server/usage/task.ts#L22) interval 1 min
- Run [Telemetry usage payload API](https://docs.elastic.dev/telemetry/collection/snapshot-telemetry#telemetry-usage-payload-api) in your browser console to verify the telemetry data looks the same
